test/common/deployment_types.go

```diff
@@ -137,13 +137,12 @@ var (
 		{
 			Name: "redhat-rhmi-fuse",
 			Products: []Product{
-				Product{Name: "broker-amq", ExpectedReplicas: 0},
-				Product{Name: "syndesis-db", ExpectedReplicas: 1},
 				Product{Name: "syndesis-meta", ExpectedReplicas: 1},
 				Product{Name: "syndesis-oauthproxy", ExpectedReplicas: 1},
 				Product{Name: "syndesis-prometheus", ExpectedReplicas: 1},
 				Product{Name: "syndesis-server", ExpectedReplicas: 1},
 				Product{Name: "syndesis-ui", ExpectedReplicas: 1},
+				{Name: "broker-amq", ExpectedReplicas: 1},
```

Comment:
```suggestion
				Product{Name: "broker-amq", ExpectedReplicas: 1},
```
Maybe it is not needed to Go.

Reply:
It's not needed, Go allows implicit typing inside slice declarations, I will add it back in to keep consistency though.

Reply:
Good to know, thanks @aidenkeating